spec/iex/resources/chart_spec.rb

```diff
@@ -72,4 +77,14 @@
       expect { subject }.to raise_error IEX::Errors::SymbolNotFoundError, 'Symbol INVALID Not Found'
     end
   end
+
+  context 'with bad request', vcr: { cassette_name: 'chart/badRequest' } do
+    subject do
+      IEX::Resources::Chart.get('MSFT', '1d', chart_interval: 10, invalid: 'option')
+    end
+
+    it 'fails with BadRequestError' do
+      expect { subject }.to raise_error IEX::Errors::BadRequestError
```

Comment:
You should be checking that the exception caught has a `.response` object and that its error message is the extracted message from the body.